Sources/App/Views/PackageController/Builds/BuildShow+View.swift

```diff
@@ -103,14 +103,6 @@ private extension Build.Status {
 
     // There should only be "ok" or "failed" builds visible on this page (others don't have any details to display) so these other cases are only covered for completeness.
```

Comment:
I believe this comment came with `var text: String`, so it should perhaps move to `description` or go away, depending on whether `description` is really identical.

Reply:
(The comment makes it sound like `text` was rather specific to this page but it might be ok to generalise.)

Reply:
I left this comment as it also applies to the two other properties directly below this. All three previously ignored the `infrastructureError`. `triggered`, and `timeout` cases, and now only two do, but the comment is still important.

Reply:
I also didn't duplicate this comment in the new implementation as the new implementation no longer ignores the other cases.

Reply:
Ah, I see, I hadn't expanded the review snippet :+1: